step.yml

```diff
@@ -43,7 +43,7 @@ inputs:
 
         **IMPORTANT**: The Scheme has to be marked as __shared__ in Xcode!
       is_required: true
-  - simulator_device: iPhone 6s Plus
+  - simulator_device: iPhone 8 Plus
```

Comment:
Is `iPhone 8 Plus` available on Xcode < 11 stacks?

Reply:
It is, excluding Xcode 8.3 stack.